modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -503,6 +505,7 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
           encryptedPrivateShare: bitGoToRecipientShare.privateShare,
           n: bitGoToRecipientShare.n!,
           vssProof: bitGoToRecipientShare.vssProof,
+          privateShareProof: bitGoToRecipientShare.privateShareProof,
```

Comment:
i was looking in the code, was this really necessary, or is it going to be needed in WP to be stored to DB?

Reply:
I think for this line, its probably just make it consistent so that anywhere the vssProof is set, the privateShareProof is also set.